package.json

```diff
@@ -29,7 +29,7 @@
         "lint:root": "pnpm prettier --check './!(demo|docs|packages|storybook)/**/*.{js,json,md,yml,yaml}' && $npm_execpath cspell .changeset/*.md",
         "lint:eslint": "pnpm --recursive run lint:eslint",
         "lint:tsc": "pnpm --recursive run lint:tsc",
-        "lint:knip": "knip --exports --files --dependencies --tags=-knipignore",
+        "lint:knip": "dotenv -- knip --exports --files --dependencies --tags=-knipignore",
```

Comment:
This is necessary to get the env variables in vite.config.mts otherwise the knip lint fails for some reason

Reply:
It would be better to do

```
proxy: process.env.API_URL_INTERNAL ?{
    "/api": {
        target: new URL(process.env.API_URL_INTERNAL).origin,
        changeOrigin: true,
        secure: false,
    },
} : undefined
```

in the Vite config.

Reply:
c9b59905adce70708938404ada1f58b24c2aea97